lib/routes/ui_test/ui_test_page.dart

```diff
@@ -173,6 +173,7 @@ class UITestPage extends StatelessWidget {
               title: const Text("QrCodeDialog"),
               onTap: () async {
                 LNInvoice invoice = LNInvoice(
+                  network: Network.Bitcoin,
```

Comment:
I'll address this on main branch, please revert this and rebase afterwards.

Reply:
Done